.github/workflows/publish-release.yml

```diff
@@ -3,6 +3,7 @@ name: release
 on:
   release:
     types: [published]
+  workflow_dispatch:
```

Comment:
- What if there is a corrupted package already?
- It will work for conda-forge also?
- I could find the issue in [conda-forge CD](https://github.com/conda-forge/petl-feedstock) yet

Reply:
- Complementing my previous comments
- It would be amazing to have such functionality
- How does it work?
  - Can I select the commit or the version?
  - It's possible to overwrite a corrupted version also?

Reply:
I haven't tried this yet to be honest. I'll try to give it a go on test.pypi.org from my branch and update you with the outcome

Reply:
- Neither I have explored it yet.
- I haven't tried before because:
  - It was the first time the release pipeline was broken.
  - It looks like it matter most using the time somewhere else
  - I didn't have the PyPI/feedstock admin access

Reply:
@arturponinski,

I think I [figured out](https://github.com/juarezr/firebirdsql-github-action/blob/master/.github/workflows/testing_changes.yml#L15) how to use `workflow_dispatch` for triggering actions.

Basically:
1. Setup `workflow_dispatch` with up to 10 `input`  tags / entries.
2. React in the workflow steps according to the `inputs` and event type
3. Github Actions will show a trigger button when this is set up accordingly.